src/utils.js

```diff
@@ -71,46 +98,55 @@ function getRelevantESLintConfig(eslintConfig) {
  */
 function getPrettierOptionsFromESLintRules(eslintConfig, prettierOptions) {
   const {rules} = eslintConfig
-  const optionGetters = {
-    printWidth: getPrintWidth,
-    tabWidth: getTabWidth,
-    parser: getParser,
-    singleQuote: getSingleQuote,
-    trailingComma: getTrailingComma,
-    bracketSpacing: getBraketSpacing,
-  }
 
-  return Object.keys(optionGetters).reduce(
-    (options, key) => {
-      const givenOption = prettierOptions[key]
-      const optionIsGiven = prettierOptions[key] !== undefined
-      const getter = optionGetters[key]
-      options[key] = optionIsGiven ? givenOption : getter(rules)
-      return options
-    },
+  return Object.keys(OPTION_GETTERS).reduce(
+    (options, key) => configureOptions(prettierOptions, key, options, rules),
     {},
   )
 }
 
-function getPrintWidth(rules) {
-  return getRuleValue(rules, 'max-len', 80, 'code')
+// If an ESLint rule that prettier can be configured with is enabled create a
+// prettier configuration object that reflects the ESLint rule configuration.
+function configureOptions(prettierOptions, key, options, rules) {
+  const givenOption = prettierOptions[key]
+  const optionIsGiven = givenOption !== undefined
+
+  if (optionIsGiven) {
+    options[key] = givenOption
+  } else {
+    const {ruleValue, ruleValueToPrettierOption} = OPTION_GETTERS[key]
+    const getterRuleValue = ruleValue(rules)
```

Comment:
You mean `getterRuleValue`? Well would it not clash with `ruleValue` from the `OPTION_GETTERS`? How about `ruleValueGetter`?